src/components/ChainSelector/ChainSelector.test.tsx

```diff
@@ -42,3 +42,5 @@ test('renders all the cards', async () => {
     expect(getByText('Ropsten')).toBeDefined();
     expect(getByText('Goerli')).toBeDefined();
 });
+
+// TODO: Use userEvent testing after React18 upgrade
```

Comment:
I had a thought when i was talking to @AbhinavMV ...

Instead of adding a TODO can you this?

```
xtest('name of test', () => {
  // TODO what to test
});
```

notice `xtest` this means the test runner adds a skipped test, this will bring our attention to it later

Reply:
Didn't know about this xtest
Nice one! thank you